_includes/documentation/v3/mediasoup/api/Router.md

```diff
@@ -285,6 +285,30 @@ const transport2 = await router2.createWebRtcTransport({ ... });
 const consumer2 = await transport2.consume({ producerId: producer1.id, ... });
 ```
 
+#### router.createActiveSpeakerObserver(options)
+{: #router-createActiveSpeakerObserver .code}
+
+Creates a new audio level observer.
+
+<div markdown="1" class="table-wrapper L3">
+
+Argument    | Type    | Description | Required | Default
+----------- | ------- | ----------- | -------- | ----------
+`options`   | [ActiveSpeakerObserverOptions](#ActiveSpeakerObserverOptions) | Options. | Yes |
```

Comment:
Are those options really required? no default values?

Reply:
(it's ok if required, just asking)

Reply:
No. I believe they are optional, but I need to make sure.